core/injected-scripts/domOverride_openShadowRoots.ts

```diff
@@ -1,13 +1,7 @@
 const closedShadows = new WeakSet<ShadowRoot>();
 const closedShadowOwners = new WeakSet<Element>();
 
-const shadowTriggerName = args.callbackName;
```

Comment:
Do we not need to delete these anymore? Before we deleted because they were added to the global scope, so could be seen

Reply:
No right now they are present in the scope of the parent function which the inner function can access, never stored in window